scripts/packages/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -2,16 +2,14 @@ $ErrorActionPreference = 'Stop'; # stop on all errors
 $packageName = 'bazel'
 
 $toolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
-$paramsText = get-content "$($toolsDir)\params.txt"
+$paramsText = ((get-content "$($toolsDir)\params.txt" | out-string) -split "`n")
 write-host "Content of $($toolsDir)\params.txt:"
 write-host $paramsText
```

Comment:
This is rendered incorrectly:

```
PS C:\Users\laszlocsomor> echo foo bar baz > a.txt ; type a.txt
foo
bar
baz
```

```
PS C:\Users\laszlocsomor> $foo = ((get-content "a.txt" | out-string) -split "`n")
PS C:\Users\laszlocsomor> write-host $($foo.GetType())
System.String[]
PS C:\Users\laszlocsomor> write-host $foo
 baz
PS C:\Users\laszlocsomor> write-host $foo[0]
foo
PS C:\Users\laszlocsomor> write-host $foo[1]
bar
PS C:\Users\laszlocsomor> write-host $foo[2]
baz
```

Maybe print the results of get-content (the object array) before converting it to a string array?